src/haddock/modules/analysis/caprieval/capri.py

```diff
@@ -203,10 +213,11 @@ def calc_lrmsd(self):
         if len(obs_chains) < 2:
             log.warning("Not enough chains for calculating lrmsd")
         else:
-            r_chain, l_chain = self.check_chains(obs_chains)
+            r_chain, l_chains = self.check_chains(obs_chains)
             r_start, r_end = chain_ranges[r_chain]
-            l_start, l_end = chain_ranges[l_chain]
-
+            l_starts = [chain_ranges[l_chain][0] for l_chain in l_chains]
+            l_ends = [chain_ranges[l_chain][1] for l_chain in l_chains]
```

Comment:
`l_chain` in `l_chains` is a bit confusing, how about `for _l in l_chains`

Reply:
good point!